### PR TITLE
[sailfish-browser] Use grid layout for tab view on large screen

### PR DESCRIPTION
**Pull request by rainemak**
On large screen
- 2 columns on portrait
- 3 columns on landscape

### CODE REVIEW

**Comment by siteshwar** [comment]
LGTM

**Comment by rojkov** [comment]
LGTM

File: src/pages/components/TabView.qml [hunk 1]
```diff
@@ -39,12 +42,15 @@ SilicaGridView {
     delegate: TabItem {
         id: tabItem
 
+        readonly property bool largeLeftMargin: index % columns === 0
+        readonly property bool largeRightMargin: columns > 1 ? index % columns === (columns - 1) : true
```
Comment (kananoja) [inline src/pages/components/TabView.qml]: "index % columns === (columns - 1)" should be enough. No need for the tertiary operation afaics.

Comment (kananoja) [inline src/pages/components/TabView.qml]: I would rename the properties to something like this:
- "largeLeftMargin" -> "firstColumn"
- "largeRightMargin" -> "lastColumn"
Maybe it's just me, but I find it easier to understand the margin bindings (below) if they were named like this.

Comment (rainemak) [inline src/pages/components/TabView.qml]: Thanks @kananoja will do suggested changes. firstColumn and lastColumn are better names.

**Comment by rainemak** [comment]
Updated as per comments.

**Comment by kananoja** [comment]
LGTM

**Pull request merged**